tasks/sync-granule/GranuleFetcher.js

```diff
@@ -320,20 +345,28 @@ class GranuleFetcher {
    * @returns {Array<Object>} returns the staged file and the renamed existing duplicates if any
    */
   async ingestFile(file, destinationBucket, duplicateHandling) {
+    let duplicateFound;
     const fileRemotePath = path.join(file.path, file.name);
     const sourceBucket = file.source_bucket;
     // place files in the <collectionId> subdirectory
     const stagingPath = S3.s3Join(this.fileStagingDir, this.collectionId);
     const destinationKey = S3.s3Join(stagingPath, file.name);
 
     // the staged file expected
-    const stagedFile = {
-      ...file,
-      filename: S3.buildS3Uri(destinationBucket, destinationKey),
-      fileStagingDir: stagingPath,
-      url_path: this.getUrlPath(file),
+    const stagedFile = removeNilProperties({
+      size: file.size,
       bucket: destinationBucket,
-    };
+      key: destinationKey,
+      source: `${file.path}/${file.name}`,
+      fileName: path.basename(destinationKey),
+      type: file.type,
+      checksumType: file.checksumType,
+      checksum: file.checksum ? file.checksum.toString() : file.checksum,
+    });
+
+    // if (file.checksum) {
+    //   stagedFile.checksum = file.checksum.toString();
+    // }
```

Comment:
This is in the main branch and I'm thinking it should be removed in that branch. Just wanted to comment on it so it's visible